NMEA/index.html

```diff
@@ -83,7 +83,7 @@ <h4 class="modal-title">Enter the stream URL</h4>
         var cUrl = window.location.href;
         var pName = window.location.pathname;
         var baseUrl = cUrl.split(pName)[0]
-		var urlComplete = baseUrl+'/api/nmea.txt?stream='+url;
+		var urlComplete = 'http://api.loklak.org/api/nmea.txt?stream='+url;
```

Comment:
Why is this URL hardcoded ? It needs to be available from any loklak server installation and should use that server's API endpoint than the main server.

@mariobehling Yes, https://github.com/sudheesh001/sudheesh001.github.io/tree/master/iot/data contains the list of data sources. I just tested with the stream url `http://sudheesh.info/iot/data/Garmin-GPS76_route.txt` seems to be failing because of Github's https:// traffic being routed to http:// loklak server. This will work from localhost and those that are CORS enabled.